app/controllers/registrations_controller.rb

```diff
@@ -13,6 +13,6 @@ def create
   private
 
   def user_params
-    params.permit(:username, :email, :password, :password_confirmation, :password_digest)
+    params.require(:registration).permit(:username, :email, :password, :password_confirmation, :password_digest)
```

Comment:
You had an unpermitted parameter that wasn't allowing the registration be completed